packages/design-system-docs/src/pages/components/IdleTimeout/_IdleTimeout.docs.scss

```diff
@@ -0,0 +1,31 @@
+/*
+Idle Timeout
+
+A component that counts down to the end of a session (behind the scenes) and shows a warning message at a set time.
+
+Style guide: components.idleTimeout
+*/
+
+/*
+`<IdleTimeout>`
+
+@react-example IdleTimeout.example.jsx
+
+@react-props IdleTimeout.tsx
```

Comment:
This type value is weird, but I don't think there's much we can do about it 😕 

<img width="1075" alt="Screen Shot 2022-02-16 at 4 00 40 PM" src="https://user-images.githubusercontent.com/7595652/154378003-b911978a-f87c-4eed-8ffc-1868152d8efd.png">